src/live-region/controller.ts

```diff
@@ -33,7 +33,6 @@ export class LiveRegionController {
     this._element.className = styles.announcer;
     this._element.setAttribute('aria-live', this.politeness);
     this._element.setAttribute('aria-atomic', 'true');
-    this._element.setAttribute('data-awsui-live-announcer', 'true');
```

Comment:
Not a blocker, out of curiosity: do you remember why we had this data attribute?

Reply:
Thanks for explaining! :)